mobile/src/main/java/com/gelakinetic/mtgfam/fragments/CardViewFragment.java

```diff
@@ -1744,6 +1742,8 @@ private String saveImage() {
                 return getString(R.string.card_view_no_image);
             }
 
+            FileOutputStream fStream = new FileOutputStream(fPath);
```

Comment:
Sometimes I get distracted from the main goal. The reasoning behind moving this was because you open the stream and have the possibility of returning before it was ever used.